Source/NSViewController.m

```diff
@@ -164,6 +186,45 @@ - (NSBundle *)nibBundle
   return _nibBundle;
 }
 
+// Dismiss
+- (void) dismissViewController: (NSViewController *)viewController
+{
+  [viewController viewDidDisappear: NO];
+}
+
+- (void) dismissController: (id)sender
+{
+  [self dismissViewController: self];
+}
+
+// NSSeguePerforming methods...
+- (void)performSegueWithIdentifier: (NSStoryboardSegueIdentifier)identifier
```

Comment:
Here and in the next methods a space after the return type is missing.

Reply:
Fixed, resolve if you agree.